src/components/requests/features.info.vue

```diff
@@ -1,16 +1,24 @@
 <template>
   <div ref="GetFeatureInfoRequest">
+    To show feature info box for a certain layer you must ensure first that the layer visibility is set to true and then proceed to send GetFeatureInfoRequest.
+    <br><br>
+    1. Set visibility of wanted layer as true.
+    <button id="btnMapModulePluginLayerVisibilityRequest" class="btn btn-primary exampleready" @click="mapLayerVisibilityRequest">MapModulePlugin.MapLayerVisibilityRequest</button>
+    <code-component snippet="channel.postRequest('MapModulePlugin.MapLayerVisibilityRequest', [492, true]);
+  channel.log('MapModulePlugin.MapLayerVisibilityRequest posted with data', [492, true]);"></code-component>
```

Comment:
Wouldn't it make more sense to make sure the layer is visible when the user enters the page (and possibly hide it when the user leaves). Probably should still tell the user why the layers change when the example is accessed.

Use `<CodeSnippet>...code here....</CodeSnippet>` instead of `<code-component` with the code in an attribute value to make it easier to read when maintaining.

Reply:
Updated these as suggested. Also added listener for layer visibility on startup.